helper.fish

```diff
@@ -1628,9 +1628,14 @@ function checkMacros
   and pushd $WORKDIR/work/ArangoDB
   or begin popd; return 1; end
 
-  set -l wrong (find lib arangod arangosh enterprise tests -name "*.cpp" -o -name "*.h" \
-    | xargs grep -P -n '# *ifdef +(WIN32|(TRI_ENABLE_|ARANGODB_USE_|USE_)MAINTAINER_MODE)') 
-  
+  if test -d client-tools
```

Comment:
I wonder if the list of directories could be put into a variable so that we need to implement the grep statement just once.